test/corpus/expressions.txt

```diff
@@ -833,40 +845,43 @@ lambda (a, b): (a, b)
 
 (module
   (expression_statement
-    (lambda
-      (lambda_parameters
-        (identifier)
-        (identifier))
-      (call
-        (identifier)
-        (argument_list
-          (binary_operator
-            (string (string_content))
-            (identifier))))))
-  (expression_statement
-    (lambda
-      (true)))
-  (expression_statement
-    (lambda
-      (lambda_parameters
-        (identifier)
-        (default_parameter
-          (identifier)
-          (identifier))
-        (list_splat_pattern
-          (identifier))
-        (dictionary_splat_pattern
-          (identifier)))
-      (identifier)))
-  (expression_statement
-    (lambda
-      (lambda_parameters
-        (tuple_pattern
-          (identifier)
-          (identifier)))
-      (tuple
-        (identifier)
-        (identifier)))))
+	(lambda
+	  (lambda_parameters
+		(identifier)
+		(identifier))
+	  (call
+		(identifier)
+		(argument_list
```

Comment:
The indentation in this assertion is inconsistent with most of the others. Could you adjust this to use 2-space indentation?

Reply:
done